src/components/NetworkNodeSelector/NetworkNodeSelectorTs.ts

```diff
@@ -142,19 +141,13 @@ export class NetworkNodeSelectorTs extends Vue {
     }
 
     public async created() {
-        // add static hardcoded nodes to harvesting list
-        const staticNodesUrls = [];
-        networkConfig[this.networkType].nodes.map((node) => staticNodesUrls.push(node.url.replace(/http:|:3000|\//g, '')));
-
         // add selected nodes
-        const currentNodeUrl = this.currentProfile.selectedNodeUrlToConnect.replace(/http:|:3000|\//g, '');
-        staticNodesUrls.push(currentNodeUrl);
+        const currentNodeUrl = this.currentProfile.selectedNodeUrlToConnect.replace(/http:|https:|:3000|:3001|\//g, '');
```

Comment:
Not sure do you still need to replace the protocol and port.

we can grab this from stat-service `apiStatus. restGatewayUrl`

Reply:
This is for displaying the currently selected node in the harvesting page when the user heads to the page, and we always need to trim these strings to be able to display it correctly.